### ISSUE TITLE
Unable to scan QR Code or Image

### ISSUE THREAD

**Issue by onnyyonn** [beemdevelopment/Aegis#538]
###### Info

* __Version__: 1.2.1.36 
* __Source__: F-Droid
* __Vault encrypted:__ Yes (with biometric unlock)
* __Device:__ Asus 6Z (Rooted)
* __Android version and ROM:__ Android 10, Havoc OS

###### Steps to reproduce

Add an entry with either "Scan QR code" or "Scan Image" option.

###### What do you expect to happen?

For "Scan QR code", it is expected to have the camera open for scanning.
For "Scan Image", it is expected the image to be processed and an entry is generated.

###### What happens instead?

For "Scan QR code", the camera freezes immediately after opening. I am unable to go back to the app, without force-closing it first.
For "Scan Image", the following error shows up: com.google.zxing.NotFoundException

**Comment by alexbakker** [comment]
> For "Scan QR code", the camera freezes immediately after opening. I am unable to go back to the app, without force-closing it first.

This is a rare issue that will hopefully be fixed in the next release as we switched to CameraX.

> For "Scan Image", the following error shows up: com.google.zxing.NotFoundException

Can you tell us a bit more about the QR code you're trying to scan? Which service are you trying to add to Aegis?

**Comment by onnyyonn** [comment]
The issue appears for every service I try to add. You can take Gitlab as an example, which is what I tried recently.

**Comment by alexbakker** [comment]
Huh. Scanning images doesn't work anymore for me either. Thanks for your report. We'll look into it.

**Comment by phillipprado** [comment]
I am having a similar issue to this too. The QR scanner function does nothing but open a view finder, and won't scan the code. I've tried this for several sources, and I get the same result everytime. It doesn't freeze or anything like that, but it just won't scan the QR. Uploading an image shows the same error message as the OP refers to.

What's interesting, is that I was able to get one QR code to scan once I first installed the app with no issue. But now, no other site is working.

App version: 1.3 from F-Droid
Device: Google Pixel 4XL (stock Android 11/non-rooted)
